decoder/vaapidecoder_vp8.cpp

```diff
@@ -518,14 +480,22 @@ YamiStatus VaapiDecoderVP8::decode(VideoDecodeBuffer* buffer)
 {
     YamiStatus status;
     Vp8ParserResult result;
+    if (!buffer || !buffer->data) {
```

Comment:
instead of duplicate code here is it possible add function like this
`flush(bool discardOutput)`
and call this flush in` void VaapiDecoderVP8::flush(void)` and here